djangocms_versioning/monkeypatch.py

```diff
@@ -102,7 +104,31 @@ def menu_renderer_cache_key(self):
     return key
 
 
+def get_wizard_success_url(self, obj, **kwargs):
+    language = kwargs.get('language', None)
+    return get_object_preview_url(obj, language)
+
+
+def get_page_wizard_success_url(self, obj, **kwargs):
+    language = kwargs['language']
+    cms_extension = apps.get_app_config('djangocms_versioning').cms_extension
+    versionable_item = cms_extension.versionables_by_grouper[obj.__class__]
+    page_content = (
```

Comment:
Any chance you can replace this with `page_content = obj.title_cache[language]`?